### DISCUSSION TITLE
Delete the legend "Powered by Flowise" when you insert the html code?

### DISCUSSION THREAD

**Discussion by claclin** [FlowiseAI/Flowise#1694] [Q&A] [answered]
Once you have inserted the html code on my website and press the buttom I see at the bottom the legend  "Powered by Flowise". Does anyone know how can I delete it?

**Comment by Jaredude** [comment]
Leon van Zyl has several excellent Flowise tutorials on YouTube, including how to remove the [Powered by Flowise](https://www.youtube.com/watch?v=4MSxi6-RYCI&list=PL4HikwTaYE0HDOuXMm5sU6DH6_ZrHBLSJ&index=13) text.

**Reply by claclin** [reply]
Yes, but If I have deployed with docker?

**Reply by Jaredude** [reply, marked as answer]
The chat client you add to your website, [FlowiseChatEmbed](https://github.com/FlowiseAI/FlowiseChatEmbed), is a completely separate repo from the Flowise server you're deploying via docker. Once you've deployed the server using docker, you would follow the instructions in Leon's video: fork the chat client repo, modify to remove the Powered by text, and use your own version of the client in your website. Full step by step instructions are in the YouTube video.

**Reply by claclin** [reply]
Thanks, I will try it.

**Comment by claclin** [comment]
fixed.
